pvr.hts/resources/language/resource.language.en_gb/strings.po

```diff
@@ -100,7 +100,15 @@ msgctxt "#30058"
 msgid "Defaults for new timers"
 msgstr ""
 
-#empty strings from id 30059 to 30099
+msgctxt "#30059"
+msgid "Scheduled recordings"
+msgstr ""
+
+msgctxt "#30060"
+msgid "Ignore duplicates (tvh 4.3+)"
```

Comment:
I'd use Tvheadend instead of tvh

Reply:
Then, the string gets to long and wont fit the settings dialog anymore - at least for more 'comprehensive' languages like French and German. And, we use 'tvh' already for another version dependent setting string.

Reply:
Okay, then I guess it's fine